### PR TITLE
Add snb validation to QA tests

### PR DESCRIPTION
**Pull request by sheldonkhall**

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -5,23 +5,27 @@ node {
...
-      }
-      stage('Init Grakn') {
-        sh 'if [ -d grakn-package ] ;  then rm -rf grakn-package ; fi'
-        sh 'mkdir grakn-package'
-        sh 'tar -xf grakn-dist/target/grakn-dist*.tar.gz --strip=1 -C grakn-package'
-        sh 'grakn-package/bin/grakn.sh start'
-      }
-      stage('Test Connection') {
-        sh 'grakn-package/bin/graql.sh -e "match \\\$x;"' //Sanity check query. I.e. is everything working?
+    withEnv([
```
Comment (fppt) [inline Jenkinsfile]: These environment variables will only be present for what is in the braces correct?

File: grakn-test/test-snb/src/validate-snb/readwrite_grakn--ldbc_driver_config--db_validation.properties [hunk 2]
```diff
@@ -0,0 +1,164 @@
+# -------------------------------------
```
Comment (fppt) [inline grakn-test/test-snb/src/validate-snb/readwrite_grakn--ldbc_driver_config--db_validation.properties]: That file name!! `readwrite_grakn--ldbc_driver_config--db_validation.properties`

File: grakn-test/test-snb/src/validate-snb/readwrite_grakn--ldbc_driver_config--db_validation.properties [hunk 3]
```diff
@@ -0,0 +1,164 @@
...
+# number of operations to generate during benchmark execution
+# LONG
+# COMMAND: -oc/--operation_count
+operation_count=100000
+
+# ************************************************************************************
+# *** non-core configuration parameters ***
+# ************************************************************************************
+
+ldbc.snb.interactive.LdbcQuery10_enable=true
```
Comment (fppt) [inline grakn-test/test-snb/src/validate-snb/readwrite_grakn--ldbc_driver_config--db_validation.properties]: Do these relate to which queries will be executed and tested?

File: Jenkinsfile (hunk 1 above)
Comment (sheldonkhall) [inline Jenkinsfile]: that's right, I didn't know whether I could nest withEnv calls so have left it for a general clean up at the end. I know there are too may hard coded paths, but again I am hoping to clean them up at the end

File: grakn-test/test-snb/src/validate-snb/readwrite_grakn--ldbc_driver_config--db_validation.properties (hunk 2 above)
Comment (sheldonkhall) [inline grakn-test/test-snb/src/validate-snb/readwrite_grakn--ldbc_driver_config--db_validation.properties]: ha ha my brain is off - I will refactor in the next pr

File: grakn-test/test-snb/src/validate-snb/readwrite_grakn--ldbc_driver_config--db_validation.properties (hunk 3 above)
Comment (sheldonkhall) [inline grakn-test/test-snb/src/validate-snb/readwrite_grakn--ldbc_driver_config--db_validation.properties]: helpfully, no...  Those are the queries it wants to try. The ones we actually are supporting are registered in the class GraknDb

**Pull request merged**